sdcm/sct_config.py

```diff
@@ -1824,6 +1825,33 @@ def _check_backend_defaults(self, backend, required_params):
         for _opt in opts:
             assert _opt['name'] in self, "{} missing from config for {}".format(_opt['name'], backend)
 
+    def _check_version_supplied(self, backend: str):
+        one_of_options_must_exist = []
+
+        if not self.get('use_preinstalled_scylla'):
+            one_of_options_must_exist += ['scylla_repo']
+        if backend == 'aws':
+            one_of_options_must_exist += ['ami_id_db_scylla']
+        elif backend == 'gce':
+            one_of_options_must_exist += ['gce_image_db']
+        elif backend == 'azure':
+            one_of_options_must_exist += ['azure_image_db']
+        elif backend == 'docker':
+            one_of_options_must_exist += ['docker_image']
+        elif backend == 'baremetal':
+            one_of_options_must_exist += ['db_nodes_public_ip']
+        elif 'k8s' in backend:
+            one_of_options_must_exist += ['scylla_version']
+        elif backend in ('aws-siren', 'gce-siren'):
+            one_of_options_must_exist += ['cloud_cluster_id']
```

Comment:
This is wrong condition. Use following:

```
elif self.params.get('db_type') == 'cloud_scylla':
```

Currently existing code where "-siren" is mentioned was kept for backwards compatibility with the `siren-tests` project, which is switched already to the usage of common `aws` and `gce` ones setting `db_type` to `cloud_scylla`. So, the latter one must be used for such a diff and the former one should be just removed from the SCT completely.

To make sure that this backend is not used look at the `sct_plugin/jenkins-pipelines` dir in `siren-tests` project.

Reply:
o.k., now fixed

but we'll need to clear those backend out of the code of SCT, it's reference more then a few times